addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -673,6 +673,23 @@ def _import_fattura_pa_line(self, element, invoice_line_form, extra_info=None):
         if line_elements:
             invoice_line_form.sequence = int(line_elements[0].text)
 
+        # Label.
+        line_elements = element.xpath('.//Descrizione')
+        if line_elements:
+            invoice_line_form.name = " ".join(line_elements[0].text.split())
+
+        # Fitting product for the line (will be replaced if there's one on the EDI)
```

Comment:
Why don't we move this code after looking at the XML first? (in the following code block up to line 715)
This way we could use prediction only if we didn't find anything, avoiding the replace, and the needless database query if we already have something valid.